.github/workflows/test-e2e.yml

```
@@ -43,6 +43,38 @@ jobs:
           path: build/
           retention-days: 1
 
+  validate-e2e-tests-without-cluster:
```

```suggestion
  validate-without-cluster:
```

Given this is the test-e2e workflow having e2e-tests in the name here stutters